src/pages/FooterPages/ContributorPage/index.jsx

```diff
@@ -0,0 +1,83 @@
+import "../design.css";
+import "./index.css";
+
+import { Box, Typography, useMediaQuery } from "@mui/material";
+import React, { useEffect, useState } from "react";
+
+import ContributorCard from "./ContributorCard";
+
+function Contributor() {
+  const [contributors, setContributors] = useState([]);
+  const isNonMobileScreen = useMediaQuery("(max-width: 800px)");
+
+  const getData = async () => {
+    const res = await fetch(
+      "https://api.github.com/repos/narayan954/dummygram/contributors?per_page=100"
```

Comment:
you're limiting this list to 100 contributors only, which works for now, but not in possible future.Need to fix this.

Reply:
Also, as a next step, we can add pagination to this

Reply:
Hey, can I try to add the pagination?

Reply:
yes, create separate issue :)

Reply:
ok thanks